### DISCUSSION TITLE
Opacity control with scroll wheel and key combination

### DISCUSSION THREAD

**Discussion by Panostis** [WindowTop/WindowTop-App#208] [Ideas]
Hello there, how about if we could control opacity holding down a key while at the same time using the scroll wheel.
Thank you!

**Comment by gileli121** [comment]
@Panostis 
Thanks for the idea.
The toolbar is not easy enough?

**Reply by Panostis** [reply]
It is. Not only that but it is very helpful indeed. Perhaps it would be faster if you could just, you know, adjust the active window's opacity with a key, scroll wheel combo. Maybe it would be a little faster not having to drive the mouse pointer to the windowtop controls. On second more mature thought Maybe you could implement a key, scroll wheel combo JUST for the active window and leave the opacity controls as they are because i really like them as well. On many occasions i find myself wanting to adjust the active window's opacity to see what's under. If i could do it with scroll wheel it would be nice. if it can be done i mean. Thanks a lot.
